datadreamer/prompt_generation/lm_prompt_generator.py

```diff
@@ -33,18 +38,22 @@ def __init__(
         num_objects_range: Optional[List[int]] = None,
         seed: Optional[float] = 42,
         device: str = "cuda",
+        quantization: Optional[str] = "none",
     ) -> None:
         """Initializes the LMPromptGenerator with class names and other settings."""
         num_objects_range = num_objects_range or [1, 3]
-        super().__init__(class_names, prompts_number, num_objects_range, seed, device)
-        self.model, self.tokenizer = self._init_lang_model()
+        super().__init__(
+            class_names, prompts_number, num_objects_range, seed, device, quantization
+        )
+        self.model, self.tokenizer, self.pipeline = self._init_lang_model()
 
     def _init_lang_model(self):
         """Initializes the language model and tokenizer for prompt generation.
 
         Returns:
             tuple: The initialized language model and tokenizer.
```

Comment:
The docstring is outdated. Also please specify the return type of the function in the function declaration as well.